src/main/java/sirius/kernel/di/transformers/AutoTransformLoadAction.java

```diff
@@ -34,25 +34,25 @@
 
     protected static class AutoTransformer<S, T> implements Transformer<S, T> {
 
-        private int priority;
-        private Class<S> sourceType;
-        private Class<T> targetType;
-        private Class<?>[] additionalTargetTypes;
-        private Class<?> transformerClass;
+        private final int priority;
```

Comment:
Wär super wenn beim nächsten PR Codestyle und echte Änderung in zwei Commits aufgeteilt wäre 😇